operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -266,8 +266,9 @@ public static Step createAbortedFailureSteps() {
    * Asynchronous steps to set Domain condition to Failed and to generate DOMAIN_FAILED event.
    *
    * @param message a fuller description of the problem
+   * @param next the next step to run. May be null.
    */
-  public static Step createTopologyMismatchFailureSteps(String message) {
+  public static Step createTopologyMismatchFailureSteps(String message, Step next) {
```

Comment:
I don't see newly added "next" is used.

Reply:
Yes, it was supposed to be passed to the FailureStep. Thanks for catching it.